jest/defs.bzl

```diff
@@ -105,21 +121,35 @@ def jest_test(
 
             See https://jestjs.io/docs/configuration#testsequencer-string for more information on Jest testSequencer config option.
 
-        snapshot_ext: The expected extensions for snapshot files. Defaults to `.snap`, the Jest default.
+        snapshots_directory: The expected snapshot directory. Defaults to `__snapshots__`, the Jest default.
+
+            This is load bearing when `snapshots` is True.
```

Comment:
should it be an error to specify snapshots_directory but not snapshots?

or perhaps you can skip this attribute. snapshots can be `true` for `__snapshots__`, or a string like `__my_snapshots__` for a folder, or a list of files

Reply:
that's true, the overloaded attribute is nice

Reply:
✅